lib/src/screens/debug/scheme_management/scheme_management_screen.dart

```diff
@@ -0,0 +1,222 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_i18n/flutter_i18n.dart';
+import 'package:rxdart/rxdart.dart';
+
+import '../../../models/enrollment_events.dart';
+import '../../../models/error_event.dart';
+import '../../../models/irma_configuration.dart';
+import '../../../models/scheme_events.dart';
+import '../../../theme/theme.dart';
+import '../../../util/combine.dart';
+import '../../../widgets/irma_app_bar.dart';
+import '../../../widgets/irma_icon_button.dart';
+import '../../../widgets/irma_repository_provider.dart';
+import '../../../widgets/progress.dart';
+import '../../../widgets/translated_text.dart';
+import '../../error/error_screen.dart';
+import '../util/snackbar.dart';
+import 'scheme_manager_detail_screen.dart';
+import 'widgets/confirm_scheme_public_key_dialog.dart';
+import 'widgets/provide_scheme_url_dialog.dart';
+import 'widgets/scheme_manager_tile.dart';
+
+class SchemeManagementScreen extends StatefulWidget {
+  @override
+  State<SchemeManagementScreen> createState() => _SchemeManagementScreenState();
+}
+
+class _SchemeManagementScreenState extends State<SchemeManagementScreen> {
+  StreamSubscription? _errorSubscription;
+
+  @override
+  void initState() {
+    super.initState();
+
+    WidgetsBinding.instance.addPostFrameCallback((_) async {
+      final repo = IrmaRepositoryProvider.of(context);
+      _errorSubscription = repo.getEvents().whereType<ErrorEvent>().listen(
+            _onErrorEvent,
+          );
+    });
+  }
+
+  void _onErrorEvent(ErrorEvent event) {
+    final navigator = Navigator.of(context);
+
+    navigator.push(
+      MaterialPageRoute(
+        builder: (context) => ErrorScreen.fromEvent(
+          error: event,
+          onTapClose: () => navigator.pop(),
+        ),
+      ),
+    );
+  }
+
+  Future<void> _onInstallScheme(BuildContext context) async {
+    final repo = IrmaRepositoryProvider.of(context);
+
+    final schemeUrl = await showDialog<String>(
+      context: context,
+      builder: (context) => const ProvideSchemeUrlDialog(),
+    );
+
+    if (schemeUrl == null) return;
+
+    String publicKey;
+    try {
+      final Uri uri = Uri.parse('$schemeUrl/pk.pem');
+      final request = await HttpClient().getUrl(uri);
+      final response = await request.close();
+      publicKey = await response.transform(utf8.decoder).first;
+      if (response.statusCode != 200) {
+        throw 'HTTP status code ${response.statusCode} received';
+      }
+    } catch (e) {
+      showSnackbar(context, 'Error while fetching scheme: ${e.toString()}.');
```

Comment:
What happend to me was that this snackbar was being triggered when I already closed the screen. This led to an exception because `context` is not usable anymore then. It would be nice if we add `if (!mounted) return` condition at these places.

Note: this comment holds for all occurences of `showSnackbar` in this PR that follows an `await.